clusters/app.ci/.diff

```diff
@@ -964,17 +964,23 @@ index a40957d..d519767 100644
        terminationGracePeriodSeconds: 300
        containers:
 diff --git a/./core-services/prow/03_deployment/boskos_reaper.yaml b/./clusters/app.ci/prow/03_deployment/boskos_reaper.yaml
-index d0b77bf..ae76eec 100644
+index f75d064..ae76eec 100644
 --- a/./core-services/prow/03_deployment/boskos_reaper.yaml
 +++ b/./clusters/app.ci/prow/03_deployment/boskos_reaper.yaml
```

Comment:
I think that in this case we need to do the same change to `./clusters/app.ci/prow/03_deployment/boskos_reaper.yaml`, not capture it in the config diff.

Reply:
Thanks for the very fast review :)

I ran "make update-app-ci" and commited the output like the error message said https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/8671/pull-ci-openshift-release-app-ci-config/4205

Reply:
Edited clusters/app.ci/prow/03_deployment/boskos_reaper.yaml